gradlew.bat

```diff
@@ -29,7 +29,7 @@ if "%DIRNAME%" == "" set DIRNAME=.
 set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%
 
-@rem Resolve any "." and ".." in APP_HOME to make it shorter.
+@rem Resolve any  and ".." in APP_HOME to make it shorter.

```

Comment:
Maybe edited per mistake? Should not be part of this commit

Reply:
No definitely no intentional change

Reply:
Can you revert this change in this file? Then let’s merge it